servicemodels/e2sm_rc/v1/e2sm-rc-ies/e2sm_rc.pb.go

```diff
@@ -1,3 +1,12 @@
+//
+//SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
+//
+//SPDX-License-Identifier: Apache-2.0
+
+////////////////////// e2sm-rc-ies.proto //////////////////////
+// Protobuf generated from /e2sm_rc.asn1 by asn1c-0.9.29
+// E2SM-RC-IEs { iso(1) identified-organization(3) dod(6) internet(1) private(4) enterprise(1) oran(53148) e2(1) version1(1) e2sm(2) e2sm-RC-IEs(3) }
+
```

Comment:
Please remove license header and SM related information (OID, etc.) - first 8 lines - this will make linters fail.